### PR TITLE
workflow

### PR DESCRIPTION
**Pull request by peter-toft-greve**

### CODE REVIEW

File: workflow [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Skip to content
+DiracResearch
```
Comment (peter-toft-greve) [inline workflow:2]: Remove line 2? It came from https://github.com/DiracResearch/dodsp/actions/runs/5387757405/workflow

File: workflow [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+/
+dodsp
```
Comment (KSAdk) [inline workflow:4]: It does not look like it is possible to highligt only a single character

File: workflow [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Code
+Issues
+Pull requests
+Actions
+Projects
+Security
+Insights
+ubuntu-22.04
+Update sonarcloud.yml #36
+Jobs
```
Comment (KSAdk) [inline workflow:17]: It is not possible to suggest a code change by clicking a button

File: workflow [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Run details
+Workflow file for this run
+.github/workflows/ubuntu-22.04.yaml at b8676be
+name: ubuntu-22.04
```
Comment (KSAdk) [inline workflow:21]: What happens if I use "Add code"
`name: ubuntu-22.04.x`

**Review by KSAdk — CHANGES_REQUESTED** [review]
I looks solid!

File: workflow (hunk 3 above)
Comment (peter-toft-greve) [reply workflow:17]: yes

File: workflow (hunk 4 above)
Comment (peter-toft-greve) [reply workflow:21]: dunno

**Review by KSAdk — APPROVED** [review]

File: workflow (hunk 1 above)
Comment (KSAdk) [reply workflow:2]: Sounds like a good idea!

**Review by KSAdk — APPROVED** [review]
Let's go!

**Pull request merged**